clients/linux/src/app.rs

```diff
@@ -603,7 +603,17 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
+                        Ok(f) => {
+                            lb.gui.win.set_title(&f.name);
+                            lb.gui.account.set_search_field_text(&lb.core.full_path_for(&f));
+                            lb.messenger.send(Msg::RefreshSyncStatus);
+                        }
+                        Err(err) => {
+                            lb.messenger.send_err("getmting renamed file", err);
```

Comment:
Let's just keep this on one line now since it's one statement, and also "getmting" -> "getting".